lib/appium.js

```diff
@@ -127,10 +127,35 @@ class AppiumDriver extends BaseDriver {
     // since we don't call super.executeCommand, we need
     // to clear the appium driver timeout manually
     this.clearNewCommandTimeout();
-
     let sessionId = args[args.length - 1];
     return this.sessions[sessionId].executeCommand(cmd, ...args);
   }
+
+  proxyActive (sessionId) {
+    let driver = this.sessions[sessionId];
+
+    // drivers need to explicitly say when the proxy is active
+    if (!driver || !driver.jwpProxyActive) {
```

Comment:
i'm confused. given that all the drivers implement MJSONWP, shouldn't _all_ the drivers (especially the ones that currently countenance `driver.jwpProxyActive`) implement `driver.proxyActive()` instead?

in other words, I'm wondering why you inspect `driver.jwpProxyActive` here instead of the sub-driver's `proxyActive()` method